File: ejercicios-js/ejercicio3.js
const palabras = ["Aceituna", "Murciélago", "Educación", "Aeropuerto", "Otorrinolaringólogo", "Euforia", "Aceite", "Paleontólogo", "Arquitectura", "Hipopótamo"];
const palabra = palabras[Math.floor(Math.random() * palabras.length)];
const vocales = palabra.toLowerCase().split('').filter(c => "aeiouáéíóú".includes(c));
console.log(`Palabra: ${palabra}`);
console.log(`Vocales: ${vocales.join(", ")}`);
console.log(`Total de vocales: ${vocales.length}`);

File: ejercicios-py/ejercicio3.py
import random

palabras = ["Aceituna", "Murciélago", "Educación", "Aeropuerto", "Otorrinolaringólogo", "Euforia", "Aceite", "Paleontólogo", "Arquitectura", "Hipopótamo"]
palabra = random.choice(palabras)
vocales = [c for c in palabra.lower() if c in "aeiouáéíóú"]
print(f"Palabra: {palabra}")
print(f"Vocales: {', '.join(vocales)}")
print(f"Total de vocales: {len(vocales)}")